platform/resource-bundle-locator/src/main/java/ddf/platform/resource/bundle/locator/ResourceBundleLocator.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.platform.resource.bundle.locator;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+/**
+ * Provides access to resource bundles for keyword internalization.
+ *
+ * <p><b> This code is experimental. While this interface is functional and tested, it may change or
+ * be removed in a future version of the library. </b>
+ */
+public interface ResourceBundleLocator {
+
+  /**
+   * Retrieves a resource bundle for a given properties file
+   *
+   * @param baseName The base name of the resource bundle
+   * @return A resource bundle for the given base name
+   * @throws java.util.MissingResourceException if the resource bundle does not exist or if an error
+   *     occurs while loading the file
+   */
+  ResourceBundle getBundle(String baseName);
```

Comment:
✏️ You should document which locale it will default to.

Reply:
✏️ Since this API actually provide access to data, it should document that it could throw a SecurityException if the specific permissions are not granted to the caller and document what those permissions are.